### PR TITLE
Milestone 4: UI

### PR DESCRIPTION
**Pull request by nicolasmarino99**
This milestone was focused on adding styles to the components. I did some extra features for making it more functional. 
#### components 🔌
- Implementation of a  functional progress bar.
- implementation of extra inputs to handle pages info.
#### Reducers 🔌
- N/A
#### Actions 🔌
- N/A.
#### Store 🔌
- N/A.

### **Styles**🎨
- Styles added to each component.

### CODE REVIEW

**Comment by nicolasmarino99** [comment]
I can't fix one stylelint problem.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # React-Redux BookStore app
 The calculator counts with:
```
Comment (suretrust) [inline README.md]: - [x] Update your readme appropriately, there are still some words like `calculator`.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
-![screenshot](./src/imgs/calculator.jpg)
+![screenshot](./src/assets/imgs/Reactapp.jpg)
 
 [Live version](https://react-calculator99.herokuapp.com)
```
Comment (suretrust) [inline README.md]: - [x] The link in your readme redirects to a calculator, let's ensure the app is available online and add the link to the readme.

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi team, 👋 

### Great job thus far! 🙌 Please see my comments. 👇 

- [x] The `ADD NEW BOOK` section of your design should have the same width as each book listed above, padding/margin inclusive. Kindly check the given design and ensure your design looks **EXACTLY** the same.

**KIndly submit another CRR once these issues have been solved.** _Happy coding!_ 🎉

**Review by shloch — APPROVED** [review]
## Hello @nicolasmarino99 


The final rendering is great
book

Happy coding

**Pull request merged**